pkg/TIPA.py

```
@@ -51,84 +74,112 @@ class ProcessManager(threading.Thread):
     Recieves instructions from the GUI via a different queue.
     '''
     def __init__(self, gui_queue, command_queue):
-        super(ProcessManager, self).__init__()
+        super(ProcessManager, self).__init__(name="ProcessManagerThread")
```

Ruff (0.8.0)

Use `super()` instead of `super(__class__, self)`

Remove `__super__` parameters

(UP008)